src/allmydata/blacklist.py

```diff
@@ -34,10 +34,10 @@ def read_blacklist(self):
         try:
             if self.last_mtime is None or current_mtime > self.last_mtime:
                 self.entries.clear()
-                with open(self.blacklist_fn, "r") as f:
+                with open(self.blacklist_fn, "rb") as f:
                     for line in f:
                         line = line.strip()
-                        if not line or line.startswith("#"):
+                        if not line or line.startswith(b"#"):
```

Comment:
I'm pretty sure you can declare victory on blacklist with this PR as well.

Reply:
I never checked test coverage, though.

Reply:
Can always do it with tiny PR later.

Reply:
> I never checked test coverage, though.

It's in `test_grid`, which is the module enabled on this PR.

> Can always do it with tiny PR later.

My speciality! ☺️ &nbsp;#955